roles/openshift_facts/library/openshift_facts.py

```diff
@@ -337,9 +337,14 @@ def normalize_provider_facts(provider, metadata):
             facts[arg] = {'cloud-provider': [provider],
                           'cloud-config': ['/etc/cloud.conf']}
 
+    if os.path.exists('/etc/aws/aws.conf'):
```

Comment:
These should probably be dependent on the existence of openshift.cloudprovider.kind instead of testing for the file on disk...